scripts/joinmarket-qt.py

```diff
@@ -698,6 +739,15 @@ def qt_directsend_callback(rtxd, rtxid, confs):
                 self.cleanUp()
             return
 
+        if bip78url:
+            manager = parse_payjoin_setup(self.bip21_uri,
+                mainWindow.wallet_service, mixdepth)
+            reactor.callWhenRunning(send_payjoin, manager,
+                accept_callback=self.checkDirectSend,
+                info_callback=self.infoDirectSend)
+            reactor.run()
```

Comment:
This won't work as it leads to `reactorAlreadyRunning` errors (for JMQT the global twisted reactor is started in `reactor.runReturn()` at the end of the file, at the start of the process). To fire a call asynchronously and then continue, use `task.deferLater(0.0, function_name, args..)` (we use it in quite a few places) instead of `callWhenRunning`.